src/modules/csm/state.py

```diff
@@ -49,6 +49,8 @@ class State:
     _epochs_to_process: tuple[EpochNumber, ...]
     _processed_epochs: set[EpochNumber]
 
+    _consensus_version: int = 1
```

Comment:
Maybe setup it on initialization in `__init__`?

Reply:
Need to check backward compatibilty first.

Reply:
`__init__` is not called upon restoring object via pickle. So we will get an `AttributeError` on access to the variable.